resources/install/debian/postinst

```diff
@@ -96,8 +96,28 @@ case "$1" in
         # and this breaks startup script
         sed -i 's/\$JVB_EXTRA_JVM_PARAMS//g' $CONFIG
 
-        if ! grep -q "org.jitsi.videobridge.AUTHORIZED_SOURCE_REGEXP" "$NEW_JITSI_CONFIG" ;then
-            echo "org.jitsi.videobridge.AUTHORIZED_SOURCE_REGEXP=focus@auth.${JVB_HOSTNAME}/.*" >> $NEW_JITSI_CONFIG
+        if ! grep -q "#org.jitsi.videobridge.AUTHORIZED_SOURCE_REGEXP" "$NEW_JITSI_CONFIG" ;then
+            sed -i 's/org.jitsi.videobridge.AUTHORIZED_SOURCE_REGEXP/#org.jitsi.videobridge.AUTHORIZED_SOURCE_REGEXP/g' $NEW_JITSI_CONFIG
+        fi
+
+        # disables tcp harvester to make sure jvb will not take port 443
```

Comment:
With the new config code switching a default looks like this :)
https://github.com/jitsi/jitsi-videobridge/pull/1030/commits/8f4e05a9d917319dcae988df08d2989fc78eebb2

Reply:
I had dropped this commit from this PR.